sections/futures/Orders/Orders.tsx

```diff
@@ -274,7 +273,16 @@ const StatusText = styled.div`
 	margin-left: 4px;
 `;
 
-const StatusIcon = styled(Svg)<{ status: OrderStatus }>`
+const StatusPendingIcon = styled(PendingIcon)<{ status: OrderStatus }>`
+	color: ${(props) =>
+		props.status === OrderStatus.PENDING
+			? props.theme.colors.yellow
+			: props.status === OrderStatus.CONFIRMED
+			? props.theme.colors.green
+			: props.theme.colors.red};
+`;
```

Comment:
Not sure how this particular one works exactly, but I think the check is redundant, since this is already the PendingIcon. You should probably be able to set the color to yellow without checking.